src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -3,11 +3,13 @@
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
+import java.util.List;
 
 public record CreateBookRequestDto(@NotBlank String title,
                                    @NotBlank String author,
                                    @NotBlank String isbn,
                                    @NotNull BigDecimal price,
                                    String description,
-                                   String coverImage) {
+                                   String coverImage,
+                                   List<Long> categoryIds) {
```

Comment:
the same. Following the business logic can the book exist without a category? (I guess no)